react/components/RadioGroup/index.js

```diff
@@ -75,6 +78,9 @@ class RadioGroup extends React.Component {
               </label>
             )
           })}
+          {errorMessage && (
+            <div className="c-danger t-small mt3 lh-title">{errorMessage}</div>
```

Comment:
Also, shouldn't it be a `span` element?

Reply:
I have used `Input` as an example to keep consistency. I think the reason for using `<div>` here is to render the error by block to not render inline with other possible components bellow.